src/Uno.UI.Runtime.Skia.Wpf/UI/Controls/UnoCompositeWindowHost.cs

```diff
@@ -97,6 +105,11 @@ private void WpfHost_Loaded(object _, System.Windows.RoutedEventArgs __)
 
 	void UpdateRendererBackground()
 	{
+		if (_renderLayer.Renderer is null)
```

Comment:
If this is supposed to happen infrequently, would adding some Log.Info / Debug be beneficial here?

Reply:
It should not happen, but it is not a problem either, the call can just be ignored